### ISSUE TITLE
Temporary files still existing

### ISSUE THREAD

**Issue by spcsser** [verkkokauppacom/parallel-phpunit#4]
First of: thanks for the great tool! It's pretty useful for running selenium tests :)

Now to the issue: I noticed that after exiting there are still some log files present created by pu. Is this a bug or a feature?

Edit: I am refering to the parallel-phpunit.\* files.

**Comment by spcsser** [comment]
(As this might be related I add this here) On exiting I noticed this unexpected output:

``` bash
/usr/bin/parallel-phpunit: line 94: 25524 Terminated              test -z "`find $ROOT -name parallel-phpunit.${PID}.*.fail`"
```

**Comment by razielsd** [comment]
fix notice here: https://github.com/razielsd/parallel-phpunit-run-by-file/commit/1352f2e9c94da6d50db7e1720be2ef2ef14212f8

**Comment by spcsser** [comment]
Thanks for the quick fix! I can't test right now, I'll test later and reply then if it solves my issue.

**Comment by spcsser** [comment]
Works like a charm, thx!

**Comment by siivonen** [comment]
You were fast! 

The log files are written next to tests and they should be removed on exit. I've seem some "bad exit cases" where
the log file deletion doesn't work.

I just commited to master a change where we use separate temporary directories instead of writing the files next to tests. So now there is no situation where parallel-phpunit.\* files are left after test execution.

I also added the fix by 'razielsd' even though I haven't been able to reproduce the problem in any of the machines that I have tried. And I added 'razielsd' to the collaborators list in case he wants to commit the fixes directly to original parallel-phpunit repository.
